libcontainer/container_linux_test.go

```diff
@@ -134,11 +143,27 @@ func (m *mockProcess) forwardChildLogs() {
 }
 
 func TestGetContainerPids(t *testing.T) {
+	pid := 1
+	stat, err := system.Stat(pid)
+	if err != nil {
+		t.Fatalf("can't state pid %d", pid)
```

Comment:
nit: `s/state/stat/`
nit: include the actual error as well

Reply:
done